index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css2?family=Material+Symbols+Sharp:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+</head>
+
+<body>
+  
+  <header>
+    <img id="header-logo" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+      alt="">
+    <ul>
+      <a href="#features">
+        <li>Features</li>
+      </a>
+      <a href="#how_it_works">
+        <li>How It Works </li>
+      </a>
+      <a href="#pricing">
+        <li>Pricing</li>
+      </a>
+    </ul>
```

Comment:
It is good practice to insert your link (<a>) tags in the (<li>) tag. This is to ensure that the link property is applied only to the <li> element.